internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -254,13 +261,44 @@ func (f *FBCRegistryPod) podForBundleRegistry(cs *v1alpha1.CatalogSource) (*core
 			},
 		})
 
-		volumeMounts = append(volumeMounts, corev1.VolumeMount{
-			Name:      k8sutil.TrimDNS1123Label(cm.Name + "-volume"),
+		if f.cfg.GzipCM {
+			volumes = append(volumes, corev1.Volume{
+				Name: k8sutil.TrimDNS1123Label(cm.Name + "-unzip"),
+				VolumeSource: corev1.VolumeSource{
+					EmptyDir: &corev1.EmptyDirVolumeSource{},
+				},
+			})
+		}
+
+		postfix := cm.Name + "-volume"
```

Comment:
Does this need the `cm.Name`? This seems unnecessary if we are also explicitly adding it on [line 279](https://github.com/operator-framework/operator-sdk/pull/6408/files?diff=split&w=0#diff-aba7d9ab4132a0f10f2e4be48a47e16124327c89a22d2de694cba5b5c22ae324R279)

```suggestion
		postfix := "-volume"
```